Resources/config/doctrine-model/Route.orm.xml

```diff
@@ -4,10 +4,10 @@
 
     <mapped-superclass name="Symfony\Cmf\Bundle\RoutingBundle\Model\Route">
         <field name="variablePattern" type="string" nullable="true"/>
-        <field name="staticPrefix" type="string" nullable="true"/>
+        <!-- we hardcode the column name to overwrite column naming strategies as we have to define the index on the column name -->
+        <field name="staticPrefix" type="string" nullable="true" column="staticPrefix"/>
 
         <indexes>
-            <index name="name_idx" columns="name"/>
```

Comment:
the name field is not defined in the Model class, only in the Doctrine\ORM class. moved it down - i hope it is possible to add indexes from several mapping files.